doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md

```diff
@@ -45,8 +45,48 @@ kubectl get all -n prometheus-system
         enabled: true
         org_role: Viewer
   ```
-
-
+* In my environment (with Ray version 2.9.0+), the Ray dashboard encountered a cross-origin issue when embedding Grafana charts. This problem was resolved after modifying the grafana.ini configuration file. Below is the complete configuration file:
+```shell
```

Comment:
I feel that separating the command and the output would be better. where the command can be formatted as `sh` and the output as `yaml`.

Reply:
corrected